.github/workflows/int_test.yml

```diff
@@ -301,9 +253,9 @@ jobs:
           azext_edge_instance: ${{ steps.env_out.outputs.INSTANCE_NAME }}
           azext_edge_init_args: ${{ steps.init.outputs.INIT_ARG }}
           azext_edge_create_args: ${{ steps.init.outputs.CREATE_ARG }}
-          azext_edge_init_redeployment: ${{ steps.init.outputs.REDEPLOY }}
+          # redeploy tested later in this workflow
+          azext_edge_init_redeployment: false
```

Comment:
tbh None/missing should be treated as false but I am fine with extra caution here